iac-extensions/arm/src/test/resources/checks/SecureParameterDefaultValueCheck/secureParameter.bicep

```diff
@@ -10,7 +10,7 @@ param nonCompliant1 string = 'my secret' // Noncompliant{{Remove the default val
 param nonCompliant2 object = {key:'my secret'} // Noncompliant{{Remove the default value from this secure object.}}
 //                           ^^^^^^^^^^^^^^^^^
 @secure()
-param nonCompliant3 string = '[newGuid()]' // Noncompliant
+param nonCompliant3 string = '[newGuid()]' // False Negative, TODO SONARIAC-1038
```

Comment:
I would like to accept this FN, as this is a super special edge case.

Reply:
Maybe also reflect in this comment or in the ticket, that FN is caused by an overlap in JSON and Bicep logic?